teps/0044-data-locality-and-pod-overhead-in-pipelines.md

```diff
@@ -301,11 +301,78 @@ controller to make this decision.
 
 ## Design proposal
 
-TBD - currently focusing on enumerating and examining alternatives before selecting one or more ways forward.
+### Summary
+
+The Tekton Pipelines controller will be updated to support running a Pipeline in a pod.
+Users will choose which parts of a pipeline to run in a pod by grouping them into a sub-Pipeline, using the [Pipelines in Pipelines](./0056-pipelines-in-pipelines.md) feature.
+Any Pipelines grouped under a Pipeline executed in a pod will also execute in that pod.
+
+We will maintain the 1-1 relationship between Tasks and TaskRuns by creating one TaskRun per Pipeline Task and using it as the source of truth for the status of the
+execution of that Task.
```

Comment:
this changes though with [matrixed pipelinetasks](https://github.com/tektoncd/community/blob/main/teps/0090-matrix.md), maybe what you wanted to say this?

```suggestion
We will create TaskRun(s) per PipelineTask and use them as the source of truth for the status of the execution of that PipelineTask.
```